webpack/assets/javascripts/react_app/components/AutoCompleteSearch/AutoComplete.js

```diff
@@ -44,13 +46,19 @@ class AutoComplete extends React.Component {
     return this.props.status === STATUS.PENDING;
   }
 
+  componentWillUnmount() {
+    this.props.resetData();
+  }
+
   render() {
     const {
       data,
       results,
       searchQuery,
       menu,
     } = this.props;
+
+    const tooltip = (<Tooltip id="__clear-tooltip__">{__('Clear')}</Tooltip>);
```

Comment:
maybe we should give it a unique id in case there are other clear tooltips on the dom?